framework/build.gradle

```diff
@@ -20,6 +20,12 @@ ext {
     apply from: 'cordova.gradle'
     cdvCompileSdkVersion = privateHelpers.getProjectTarget()
     cdvBuildToolsVersion = privateHelpers.findLatestInstalledBuildTools()
+    if (project.hasProperty('cdvMinSdkVersion')) {
+        cdvMinSdkVersion = Integer.parseInt('' + cdvMinSdkVersion)
```

Comment:
AFAIK Gradle's syntax is simply `cdvMinSdkVersion as int`

Reply:
Yes nice. I think it is time for me, to pick up a Gradle/Groovy tutorial.